docs/developer-guide/syntax-reference.rst

```diff
@@ -2093,70 +2093,74 @@ convention is followed.
-|                        |                           |            | the order they were originally processed.                           |
-+------------------------+---------------------------+------------+---------------------------------------------------------------------+
-| AVERAGE                | ``AVG(col1)``             | Stream,    | Return the average value for a given column.                        |
-|                        |                           | Table      | Note: rows where ``col1`` is null are ignored.                      |
-+------------------------+---------------------------+------------+---------------------------------------------------------------------+
-| MAX                    | ``MAX(col1)``             | Stream     | Return the maximum value for a given column and window.             |
-|                        |                           |            | Note: rows where ``col1`` is null will be ignored.                  |
-+------------------------+---------------------------+------------+---------------------------------------------------------------------+
-| MIN                    | ``MIN(col1)``             | Stream     | Return the minimum value for a given column and window.             |
-|                        |                           |            | Note: rows where ``col1`` is null will be ignored.                  |
-+------------------------+---------------------------+------------+---------------------------------------------------------------------+
-| SUM                    | ``SUM(col1)``             | Stream,    | Sums the column values                                              |
-|                        |                           | Table      | Note: rows where ``col1`` is null will be ignored.                  |
-+------------------------+---------------------------+------------+---------------------------------------------------------------------+
-| TOPK                   | ``TOPK(col1, k)``         | Stream     | Return the Top *K* values for the given column and window           |
-|                        |                           |            | Note: rows where ``col1`` is null will be ignored.                  |
-+------------------------+---------------------------+------------+---------------------------------------------------------------------+
-| TOPKDISTINCT           | ``TOPKDISTINCT(col1, k)`` | Stream     | Return the distinct Top *K* values for the given column and window  |
-|                        |                           |            | Note: rows where ``col1`` is null will be ignored.                  |
-+------------------------+---------------------------+------------+---------------------------------------------------------------------+
++------------------------+----------------------------+------------+---------------------------------------------------------------------+
```

Comment:
I know it's confusing, but the .rst docs are deprecated for new content, so if you could add this change to the [Aggregate Functions](https://github.com/confluentinc/ksql/blob/master/docs-md/developer-guide/ksqldb-reference/aggregate-functions.md) topic in the markdown docs, that would be great.

Reply:
Ack, I've reverted changes to the old docs and added an entry in the new one